### PR TITLE
EES-632 Test: Fix methodology UI tests

### PR DESCRIPTION
**Pull request by markhiveit**

### CODE REVIEW

File: tests/robot-tests/tests/general_public/methodology_page_absence.robot [hunk 1]
```diff
@@ -19,12 +19,12 @@ Navigate to Pupil absence in schools in England methodology page
 Should only be one link to absence methodology page
     [Documentation]  DFE-1359
     [Tags]  HappyPath
-    ${count}=  get element count  xpath://a[text()="Pupil absence statistics: methodology"]
+    ${count}=  get element count  xpath://ul/h3[text()="Pupil absence statistics: methodology"]
```
Comment (duncan-at-hiveit) [inline tests/robot-tests/tests/general_public/methodology_page_absence.robot]: Can you make use of the Python changes you already made for "page contains methodology link" here too?

File: tests/robot-tests/tests/general_public/methodology_page_absence.robot [hunk 2]
```diff
@@ -19,12 +19,12 @@ Navigate to Pupil absence in schools in England methodology page
...
     should be true   ${count} == 1
 
 User navigates to absence methodology page
     [Tags]  HappyPath
-    user clicks link    Pupil absence statistics: methodology
+    user clicks element     xpath://ul/h3[text()="Pupil absence statistics: methodology"]/../div/a[text()="View methodology"]
```
Comment (duncan-at-hiveit) [inline tests/robot-tests/tests/general_public/methodology_page_absence.robot]: Can you make use of the Python changes you already made for "page contains methodology link" here too?

File: tests/robot-tests/tests/general_public/methodology_page_absence.robot (hunk 1 above)
Comment (markhiveit) [reply tests/robot-tests/tests/general_public/methodology_page_absence.robot]: If I remember correctly, this test was added in response to a bug where the same methodology was appearing multiple times. I could add a check for the methodology being the only one to the python keyword, but I'm inclined to leave it as a separate test so it flags if that bug specifically reoccurs.

File: tests/robot-tests/tests/general_public/methodology_page_absence.robot (hunk 2 above)
Comment (markhiveit) [reply tests/robot-tests/tests/general_public/methodology_page_absence.robot]: I've created a new "user clicks methodology link" keyword to abstract this away, and added a "user checks methodology link" check before I call it

**Pull request merged**